hooks/persistence-elastic/Makefile

```diff
@@ -37,7 +37,7 @@ dashboard-importer-kind-import:
 
 deploy: ## 💾 Deploy this module via HelmChart into namespace "integration-tests"
 	@echo ".: 💾 Deploying '$(name)' $(hook-prefix) HelmChart with the docker tag '$(IMG_TAG)' into kind namespace 'integration-tests'."
-	helm -n integration-tests upgrade --install $(name) . --timeout 10m --wait  \
+	helm -n integration-tests upgrade --install $(name) . --timeout 20m --wait  \
```

Comment:
😂

Reply:
The problem right now is that the deployment of persistence-elastic locally takes at least 18min. I set the timeout 20min to see if that's enough. But it doesn't seem like it. (so expect another similar commit :joy:) This probably has to be moved to a nightly build.

Reply:
@Ilyesbdlala do you know why this is taking so long?

```bash
helm upgrade --install persistence-elastic secureCodeBox/persistence-elastic --wait
```

Takes 3 min on my local machine, and 2 min on the second run (once the images were cached)

Reply:
@J12934  That was the case for me until I deleted my kind cluster and created a new one to have the same initial state as the CI. The first run takes about 18 min, the next one is less than 1 minute ( after a `helm delete persistence-elastic`)
The problem is reproducible through the following:
```bash
kind delete cluster
kind create cluster

# install SCB
helm repo add secureCodeBox https://charts.securecodebox.io
kubectl create namespace securecodebox-system
helm --namespace securecodebox-system upgrade --install securecodebox-operator secureCodeBox/operator

# install persistence-elastic
cd hooks/persistence-elastic
make docker-build
make docker-export
make kind-import
make deploy # this is where the issue is.
```

Reply:
> The problem is reproducible through the following:
> 
> ```shell
> kind delete cluster
> kind create cluster
> 
> # install SCB
> helm repo add secureCodeBox https://charts.securecodebox.io
> kubectl create namespace securecodebox-system
> helm --namespace securecodebox-system upgrade --install securecodebox-operator secureCodeBox/operator
> 
> # install persistence-elastic
> cd hooks/persistence-elastic
> make docker-build
> make docker-export
> make kind-import
> make deploy # this is where the issue is.
> ```

Reply:
> > The problem is reproducible through the following:
> > ```shell
> > kind delete cluster
> > kind create cluster
> > 
> > # install SCB
> > helm repo add secureCodeBox https://charts.securecodebox.io
> > kubectl create namespace securecodebox-system
> > helm --namespace securecodebox-system upgrade --install securecodebox-operator secureCodeBox/operator
> > 
> > # install persistence-elastic
> > cd hooks/persistence-elastic
> > make docker-build
> > make docker-export
> > make kind-import
> > make deploy # this is where the issue is.
> > ```
> 
> For me, it took only about 3-5 min for make deploy to run locally (both on the main branch and the PR branch).. I had to execute `kubectl create namespace integration-tests` and `cd hook-sdk/nodjs` `make docker-build` in addition to the steps above. I checked by executing `kubectl -n integration-test get pods` (persistence-elastic and elasticsearch-master are in state _Running_). OS: Ubuntu 20.04

Yeah, it seems my theory, that the error is due to the amount of time required for persistence elastic to deploy, is false.
The error log from the CI is not helpful. I'm trying to figure out if there's a flag I can set to allow for more debug logs.

Reply:
The pods have been pending over the entire time, meaning that something is missing for kubernetes to start the pods (see the Inspect Post Failure step: https://github.com/secureCodeBox/secureCodeBox/runs/6286731595?check_suite_focus=true#step:27:31)

I've added a `kubectl describe pods` over the integration-tests namespace. This prints out the detailed output for the pods, which should tell us the reason why kubernetes can't start the pods.

Reply:
The hosted github action runners are apperently too small to run elasticsearch...
https://github.com/secureCodeBox/secureCodeBox/runs/6294995302?check_suite_focus=true#step:27:228